### PR TITLE
Add logging flag

### PR DESCRIPTION
**Pull request by adriantpaez**
Currently, some services have `json-file` as the logging driver, and others not, to improve logging driver configuration the `--logging` flag is added. If `--logging` flag is not `none` the logging section is generated for each service, using the provided value as the driver name. The default value is `json`.

### Generate without logging section
```
sedge cli --logging none
```

### Generate with `json-file` logging driver
```
sedge cli
```
or
```
sedge cli --logging json
```

## Changes:
- New flag: `--logging`
- Update the logging section in all service templates

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Testing
**Requires testing**

- [ ] Yes
- [ ] No

**In case you checked yes, did you write tests?**

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by cbermudez97 — APPROVED** [review]

File: configs/logging.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package configs
```
Comment (AntiD2ta) [inline configs/logging.go:1]: Please add a description to these functions, check other functions in the same package as reference

**Review by AntiD2ta — CHANGES_REQUESTED** [review]
LGTM, just minor changes requested. Also I think mevboost services is missing a logging driver conf as well.

**Comment by adriantpaez** [comment]
> LGTM, just minor changes requested. Also I think mevboost services is missing a logging driver conf as well.

Please check [these lines](https://github.com/NethermindEth/sedge/blob/feature/logging-flag/templates/services/docker-compose_base.tmpl#L28-L33). Is there somewhere else I need to make changes for mevboost?

File: configs/logging.go (hunk 1 above)
Comment (adriantpaez) [reply configs/logging.go:1]: Done -> https://github.com/NethermindEth/sedge/pull/154/commits/90c1b86ffaa39c5b2228a65a0da873e5ff8062cc

**Review by AntiD2ta — APPROVED** [review]

**Pull request merged**